src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -49,28 +69,35 @@
 @Api(value = "/docker.repo")
 @Produces(MediaType.APPLICATION_JSON)
 public class DockerRepoResource {
+
+    private final UserDAO userDAO;
     private final TokenDAO tokenDAO;
     private final ContainerDAO containerDAO;
     private final HttpClient client;
     public static final String TARGET_URL = "https://quay.io/api/v1/";
-    
+
     private static final ObjectMapper MAPPER = Jackson.newObjectMapper();
```

Comment:
Take a look and see if you can use dropwizard's ObjectMapper rather than creating your own.